src/components/NcEmptyContent/NcEmptyContent.vue

```diff
@@ -121,57 +121,73 @@ export default {
 ```
 </docs>
 
+<script setup lang="ts">
+import type { Slot } from 'vue'
+
+withDefaults(defineProps<{
+	/**
+	 * A header message about an empty content shown.
+	 *
+	 * @example 'No comments'
+	 */
+	name?: string
+
+	/**
+	 * Desription of the empty content
+	 *
+	 * @example 'No comments yet, start the conversation!'
+	 */
+	description?: string
+}>(), {
+	description: '',
+	name: '',
```

Comment:
Can we keep the same order as in definition: `<{ name, description }>`?
Or order it alphabetically there

Reply:
@ShGKme you mean https://vuejs.org/guide/components/props.html#reactive-props-destructure ?

Reply:
@Antreesy fixed